workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/EntityGameObjectLinker.cs

```diff
@@ -23,6 +26,18 @@ public EntityGameObjectLinker(World world, WorkerSystem worker)
             entityManager = world.GetExistingManager<EntityManager>();
         }
 
+        /// <summary>
+        ///     Links a GameObject to an ECS Entity.
+        /// </summary>
+        /// <remarks>
+        ///     All <see cref="UnityEngine.Component"/>s on the GameObject will be inserted onto the ECS Entity.
```

Comment:
need to add an remark that if a MonoBehaviour exists multiple times on a gameobject, only the first occurence will be inserted onto the ecs entity

Reply:
ping on that one and active voice :P